src/schemas/json/typst.json

```diff
@@ -0,0 +1,245 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/typst.json",
+  "title": "Typst Package Manifest",
+  "description": "Schema for typst.toml package manifest files, used by the Typst package manager and universe.",
+  "type": "object",
+  "required": ["package"],
+  "properties": {
+    "package": {
+      "type": "object",
+      "description": "Contains metadata about the package.",
+      "required": [
+        "name",
+        "version",
+        "entrypoint",
+        "authors",
+        "license",
+        "description"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The package's identifier in its namespace. Should use kebab-case if it contains multiple words. Should not contain 'typst'.",
```

Comment:
If markup is allowed here, and you want to design this manifesto around guidelines for Typst Universe submission, it would be nice to have a link to the naming guidelines which are a bit longer than that.